src/analysis/Environments/UnsupportedWorkspaceCell.tsx

```diff
@@ -1,18 +1,20 @@
 import { TooltipTrigger } from '@terra-ui-packages/components';
-import { div, h } from 'react-hyperscript-helpers';
+import { useThemeFromContext } from '@terra-ui-packages/components';
+import React from 'react';
 import { icon } from 'src/components/icons';
-import colors from 'src/libs/colors';
 
 // These are for calling attention to resources that are most likely linked to GCP v1 workspaces.
 // Rawls will no longer return v1 workspaces, but Leo does not have a way to filter out disks/cloud environments related to them.
 export const unsupportedDiskMessage =
   'This disk is not associated with a supported workspace. It is recommended that you delete it to avoid additional cloud costs.';
 export const unsupportedCloudEnvironmentMessage =
   'This cloud environment is not associated with a supported workspace. It is recommended that you delete it to avoid additional cloud costs.';
-export const UnsupportedWorkspaceCell = ({ status, message }) =>
-  div(
-    {
-      style: {
+export const UnsupportedWorkspaceCell = ({ status, message }) => {
```

Comment:
Preferably, all components/functions would have props/argument types.

```suggestion
export interface UnsupportedWorkspaceCellProps {
  status: string;
  message: string;
};

export const UnsupportedWorkspaceCell = (props: UnsupportedWorkspaceCellProps): ReactNode => {
  const { status, message } = props;
```